guides/user/pipeline/expressions/index.md

```diff
@@ -10,492 +10,356 @@ redirect_from:
-You can access values of parameters within expressions by referencing the `parameters` array. This is a shortcut for `execution.parameters`. For example, `${ parameters['run canary'] }` will evaluate to the value of the parameter called *run canary*. 
-
-### Trigger values
-
-You can reference values in the pipeline trigger via the `trigger` property. This value is available in the execution JSON under the trigger object. 
-
-The `trigger.buildInfo` field in jenkins triggers will contain details about the jenkins execution. To access the version hash of the git commit in Jenkins, for example, we can ask for `${ trigger.buildInfo.scm[0]['sha1'] }`.
-
-### Property files
-
-A feature of the Jenkins trigger is the ability to specify a properties file. Spinnaker will read the contents of this artifact ( it can be a Java .properties file, yml or json ) and add it into the pipeline context. 
-
-To specify a property file, make sure you're archiving it in your jenkins job, then provide the name of the file, i.e `mypropertyfile.properties` ( not the absolute path i.e, `workspace/build/mypropertyfile.properties` ).
-
-![](10.png)
-
-The contents of this file will now be available as a map under the trigger and accessible via `trigger.properties`. 
-
-For example, if my property file looked like this:
+You also can interpolate expressions and strings. For example if
+`expressionA` evaluates to _Hello_ and `expressionB` evaluates to _world_,
 
 ```
-BUILD_STACK=bojack
-BUILD_ANIMAL=horseman
+ ${expressionA}-randomString-${expressionB}
 ```
 
-I can then access the value from the property file as `${ trigger.properties['BUILD_ANIMAL']}` to get `horseman`.
-
-If you're adding property files from a Jenkins or Script *STAGE* ( not Trigger ), then you can omit the `trigger.properties` field as the stage. Any stage that runs after will be able to access these parameters as normal context fields. If we created a new stage and ran the same Jenkins job as above, then `${ BUILD_ANIMAL }` will resolve to `horseman`.
-
-> **Using the same field names**
->
-> If you have multiple Jenkins jobs, they can clobber the same properties. A way around this is to reference the particular stage. `${ #stage( 'first jenkins job' )['context']['BUILD_ANIMAL'] }`
-
-### Manual Judgment choices
-
-The manual judgment stage allows you to specify options when you approve a stage. Under Judgement Inputs, you can enter a choice.
-
-![](12.png)
+evaluates to _Hello-randomString-world_.
 
-When your stage executes, you will see a dropdown with the provided choice. 
+## How do I use a pipeline expression?
 
-![](13.png)
+### Where can I use pipeline expressions?
 
-In subsequent stages, you can reference the choice provided via the [#judgment( stage Name ) ](#judgmentstring) helper method `${ #judgment( 'manual judgment stage name' )}`. 
+You can use pipeline expressions in any place in the UI where you can enter
+free-form text, with the exception of the pipeline configuration stage. If you
```

Comment:
I would capitalize Configuration here, because the stage name is capitalized in the UI. And again below where you mention the Configuration stage.